common/app/views/fragments/immersiveMainMedia.scala.html

```diff
@@ -8,11 +8,13 @@
 @defining((
     page.item.tags.isUSMinuteSeries,
     page.item.elements.hasMainEmbed,
-    page.item.fields.main.nonEmpty
-)) { case (isMinuteSeries, hasEmbed, hasMainMedia) =>
+    page.item.fields.main.nonEmpty,
+    page.item.tags.isLabourLiverpoolSeries
```

Comment:
As discussed in person, this maybe should be its own template, not added to this one.